docs/src/references/c8y-log-management.md

```diff
@@ -6,46 +6,46 @@ sidebar_position: 7
 
 # Log file management from Cumulocity
```

Comment:
This doc needs a thorough update beyond just the plugin name change, to reflect the new cloud agnostic contracts instead of the c8y specific ones mentioned here.

Reply:
Another document named `tedge-log-plugin.md` exists already with the updated contracts. It's currently marked as draft. So, we just need to update that draft status and can just delete this doc, or add a deprecated flag to it until we fully remove that plugin.

Reply:
Done with https://github.com/thin-edge/thin-edge.io/pull/2213/commits/09f8e40fc990560635484e97809a35477465459e.